OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -41,14 +41,16 @@ public void Killed(Actor self, AttackInfo e)
 
 			var valued = self.Info.TraitInfoOrDefault<ValuedInfo>();
 
-			// Default experience is 100 times our value
 			var exp = info.Experience >= 0
 				? info.Experience
-				: valued != null ? valued.Cost * 100 : 0;
```

Comment:
Started the upgrade rule, but must check if 
a) I am doing this the right way
b) How to replace rules in a list.

Could you look at the rule and shout if I started in the wrong way?